### ISSUE TITLE
Example for node and graph level regression

### ISSUE THREAD

**Issue by hkim716** [rusty1s/pytorch_geometric#2078]
I'm looking for an example code that I would like to study for my own problem. 

Let say I have a 3D geometry that is formed in 3D meshes. 

I have 100 number of graph data that have 10,000 nodes and 20,000 edges. 
Each node has a scalar value `-1<**x**<1` as feature, and each edge has their geometry properties as edge features. (xyz for example)
Each graph data has their own property as a vector, and each graph has different set of node values on the meshes. 

I would like to create a model that predicts node values depending on the graph level vectors. 
I've seen a lot of examples that predict discrete values such as classification. e.g. MNIST predicts digits. 
However, I would like to ask if there is an example code for prediction of continuous values, in other words, regression?
Also, I would like to create a model not only for a node-level , but for graph-level prediction as well. 
Any comment would help a lot. I would appreciate it. 

Thanks!

**Comment by rusty1s** [comment]
In case you want to perform regression, the switch from classification usually just requires leveraging a different loss function, such as MSE or MAE loss.
Since you want to predict node values dependent on graph vectors and mesh data, a good idea might be to simply replicate the graph data and add it to each node feature vector. Then you can perform standard message passing between connected nodes, as we do [here](https://github.com/rusty1s/pytorch_geometric/blob/master/examples/faust.py).
